tests/System/FTSGraphTests.py

```diff
@@ -1,82 +0,0 @@
-########################################################################
```

Comment:
it tests mechanics for FTS2, so for us it doesn't matter no, I don't know about others.